libs/domain/site/navigation-button/stories/Static/common.ts

```diff
@@ -51,3 +52,31 @@ export const renderVariants = (): TemplateResult => html`
     { hideYAxisName: true }
   )}
 `;
+
+export const applyPseudoClasses = async (obj: {
+  canvasElement: HTMLElement;
+}): Promise<void> => {
+  await Promise.all([
+    customElements.whenDefined('oryx-site-navigation-button'),
+    customElements.whenDefined('oryx-button'),
+  ]);
+
+  obj.canvasElement
+    .querySelectorAll<HTMLElement>('.pseudo-parent')
+    .forEach((parent) => {
+      const pseudo = parent.dataset.pseudo;
+      const button = parent.querySelector<NavigationButtonComponent>(
+        'oryx-site-navigation-button'
+      );
+
+      if (pseudo) {
+        setTimeout(() => {
```

Comment:
Why do we need setTimeout here?

Reply:
Just to be sure that shadowRoot is ready. Components renders children asynchronously, and usually it's enough just to add setTimeout(0). Not the ideal solution, but suitable for storybook